Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md

```diff
@@ -0,0 +1,79 @@
+{{{
+  "title": "Partner Cloud Integration: CenturyLink AWS Reserved Instance Strategy",
+  "date": "2-7-2018",
+  "author": "Andy Watson",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+### Overview
+
+CenturyLink [Cloud Application Manager's](https://www.ctl.io/cloud-application-manager/) Cloud Optimization offers [consolidated billing](https://www.ctl.io/knowledge-base/cloud-application-manager/cloud-optimization/partner-cloud-integration-consolidated-billing/) to make payments easy. Part of the consolidated billing involves CenturyLink's strategy for billing Amazon Web Services [Reserved Instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-reserved-instances.html).
```

Comment:
Change second link to `(partner-cloud-integration-consolidated-billing.md)`

Reply:
Update completed.